src/cpp/nonkeyedcommand.cpp

```diff
@@ -26,81 +26,46 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef SMARTREDIS_CPP_COMMANDREPLYPARSER_H
-#define SMARTREDIS_CPP_COMMANDREPLYPARSER_H
+#include "nonkeyedcommand.h"
+#include "redisserver.h"
 
-#include <vector>
-#include "commandreply.h"
-#include "tensorbase.h"
-#include "enums/cpp_tensor_type.h"
+using namespace SmartRedis;
 
-using parsed_reply_map = std::unordered_map<std::string, std::string>;
-using parsed_reply_nested_map = std::unordered_map<std::string, parsed_reply_map>;
-
-namespace SmartRedis {
-
-namespace CommandReplyParser {
-
-inline std::vector<size_t> get_tensor_dims(CommandReply& reply)
+void NonKeyedCommand::set_exec_address_port(std::string address,
+                                    uint16_t port)
 {
-    /* This function will fill a vector with the dimensions of the
-    tensor.  We assume right now that the META reply is always
-    in the same order and we can index base reply elements array;
-    */
-
-    if(reply.n_elements() < 6)
-        throw std::runtime_error("The message does not have the "\
-                                "correct number of fields");
-
-    size_t n_dims = reply[3].n_elements();
-    std::vector<size_t> dims(n_dims);
-
-    for(size_t i=0; i<n_dims; i++) {
-        dims[i] = reply[3][i].integer();
-    }
-
-    return dims;
+    this -> _address = address;
+    this -> _port = port;
+    return;
 }
 
-inline std::string_view get_tensor_data_blob(CommandReply& reply)
+std::string NonKeyedCommand::get_address()
 {
-    /* Returns a string view of the data tensor blob value
-    */
-
-    //We are going to assume right now that the meta data reply
-    //is always in the same order and we are going to just
-    //index into the base reply.
-
-    if(reply.n_elements() < 6)
-        throw std::runtime_error("The message does not have the "\
-                                "correct number of fields");
-
-    return std::string_view(reply[5].str(), reply[5].str_len());
+    return this -> _address;
 }
 
-inline TensorType get_tensor_data_type(CommandReply& reply)
+uint16_t NonKeyedCommand::get_port()
 {
-    /* Returns a string of the tensor type
-    */
+    return this -> _port;
+}
 
-    //We are going to assume right now that the meta data reply
-    //is always in the same order and we are going to just
-    //index into the base reply.
+CommandReply AddressAtCommand::runme(RedisServer * r)
+{
+    return r->run(*this);
+}
 
-    if(reply.n_elements() < 2)
-        throw std::runtime_error("The message does not have the correct "\
-                                "number of fields");
+CommandReply AddressAnyCommand::runme(RedisServer * r)
+{
+    return r->run(*this);
+}
 
-    return TENSOR_TYPE_MAP.at(std::string(reply[1].str(),
-                                            reply[1].str_len()));
+CommandReply DBInfoCommand::runme(RedisServer *r)
```

Comment:
For ``DBInfoCommand`` and ``ClusterInfoCommand`` do we need to implement ``runme``?  It seems like the execution would be the same as the parent class, so we can just rely on the ``virtual`` parent class definition?